packages/near-membrane-base/src/intrinsics.ts

```diff
@@ -140,9 +141,10 @@ export function linkIntrinsics(env: VirtualEnvironment, blueGlobalThis: typeof g
     }
 }
 
-export function getFilteredEndowmentDescriptors(endowments: object): PropertyDescriptorMap {
+export function getFilteredGlobalObjectShapeDescriptors(endowments: object): PropertyDescriptorMap {
```

Comment:
I think this does not need to be exported anymore.

Reply:
I noticed that you used this. in some unit tests, but I believe that should not be sufficient reason to have this exported, and we can probably get away with the other one instead.

Reply:
Fixed